### ISSUE TITLE
[Token Update] Add navigation / bottom / tab bars component tokens

### ISSUE THREAD

**Issue by pylapp** [Orange-OpenSource/ouds-ios#1169]
### Type of tokens

- [x] Raw token (i.e. associated to _SwiftUI_ or _Swift_ value)
- [x] Semantic token (i.e. pointing to a raw token)
- [x] Component token (i.e. pointing to a semantic token, used for a component)

### Description

Tokens are missing for navigation components, there are not exposed yet to iOS codebase.
These tokens must be implemented for the components so as to complete #1135

### Types of change

- [x] Add new tokens
- [ ] Update existing tokens (i.e. change name or value)
- [ ] Remove existing token

### Checklist for Core Team

- [ ] Tokens have been updated
- [ ] Documentations have been updated
- [ ] Unit tests have been updated
- [ ] Retrocompatibility has been checked, or broken API have been flagged as deprecated with a workaround

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.22.0**
- Build number: **1763590034**
- Build tag: **187365f**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.22.0**
- Build number: **1763762836**
- Build tag: **89024b2**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.22.0**
- Build number: **1764194864**
- Build tag: **c0bc4a1**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_
